.github/workflows/pull-request.yaml

```diff
@@ -7,6 +7,24 @@ on:
       - develop
 
 jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version:
+          - 3.6
+          - 3.7
+          - 3.8
+          - 3.9
+    container: python:${{ matrix.python-version }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install
+        run: |
+          pip install -r dev-requirements.txt 
+          python3 -m pip install .
```

Comment:
@ethanknox-palmetto This does not feel right, but untangling the imports will also be a nightmare. Maybe the least-worst option for now?

Reply:
nope this is absolutely necessary current-state. we will need to re-name a bunch of things to do relative imports and then get palm's plugin install to play with it, not something I'm into atm personally.